### ISSUE TITLE
How to resize canvas element?

### ISSUE THREAD

**Issue by hyunsooda** [liabru/matter-js#448]
Hello. I'm student of korea.
I have seen your new demo site. it looks like terrific than before one.
I wonder how to resize canvas element.
I tried with following code, but render.context.canvas is read-only property.

const cvs = render.context.canvas;
            const body = document.querySelector('body');
            body.removeChild(cvs);        
            
            render.context.canvas = document.createElement('canvas');
            render.context.canvas.width = document.documentElement.clientWidth;
            render.context.canvas.height = document.documentElement.clientHeight;

I'm so sorry, but I have more one question.
I wonder How to control(such as moving) body without mouseConstraint ?
I have considered applyForce method but, it was not smooth.

I will wait to your response. 
Thank you

**Comment by liabru** [comment]
I think your code should be using `render.canvas` instead like this:

```js
render.canvas.width = document.documentElement.clientWidth;
render.canvas.height = document.documentElement.clientHeight;
```

> I wonder How to control(such as moving) body without mouseConstraint ?

You should look at the [Body Manipulation](http://brm.io/matter-js/demo/#manipulation) example which shows a few different ways to move bodies.

**Comment by hyunsooda** [comment]
Thanks for things which useful demo
I solved problems that I upload before about 1 week.  It was my mistake.
There were made problem after when I watched demo which you provided.

1.
 Body.setVelocity(bodyB, { x: 0, y: -10 });
Above code is completely work with same following code.

  //bodyB.force = { x : 0, y : -0.09}

what is different between two of code?


2.
Body.setAngularVelocity(bodyD, 0.05);

why receive only one parameter? 
I guess that function have to  receive two of parameter, if so  I will be able to set velocity follow time change.

3.
A :  Body.rotate(compound, 0.02);
B :  Body.setAngularVelocity(compound, 0.05);

what is different between A and B ?

4. 
Lastly, I hope watching that rigid body character demo which using base on matter.js 
I watched matter.js’s codepen. but, there is seems to be none character model which usable in game or animation realistically
so, I wonder if I can see these demo where.

Thank you.
